python/py-crossbar/Portfile

```diff
@@ -0,0 +1,194 @@
+                    size    382515
+
+python.versions     38
+
+# Patches:
+#
+#   patch-requirements-min.txt.diff:
+#       remove maximum version limit for idna and urllib3
+#
+#   patch-setup.py.diff:
+#       avoid installing LICENSE files into the Python site-packages root
+#
+patchfiles          patch-requirements-min.txt.diff \
+                    patch-setup.py.diff
+
+if {${name} ne ${subport} && ${name} ne "crossbar"} {
+
+    depends_build-append \
+                    port:py${python.version}-setuptools
+
+    depends_lib-append \
+                    port:py${python.version}-attrs \
+                    port:py${python.version}-autobahn \
+                    port:py${python.version}-bcrypt \
+                    port:py${python.version}-bitstring \
+                    port:py${python.version}-cbor \
+                    port:py${python.version}-click \
+                    port:py${python.version}-constantly \
+                    port:py${python.version}-cryptography \
+                    port:py${python.version}-h2 \
+                    port:py${python.version}-idna \
+                    port:py${python.version}-incremental \
+                    port:py${python.version}-jinja2 \
+                    port:py${python.version}-lmdb \
+                    port:py${python.version}-mistune \
+                    port:py${python.version}-netaddr \
+                    port:py${python.version}-passlib \
+                    port:py${python.version}-priority \
+                    port:py${python.version}-psutil \
+                    port:py${python.version}-pyubjson \
+                    port:py${python.version}-asn1 \
+                    port:py${python.version}-asn1-modules \
+                    port:py${python.version}-pygments \
+                    port:py${python.version}-pynacl \
+                    port:py${python.version}-openssl \
+                    port:py${python.version}-pyqrcode \
+                    port:py${python.version}-pytrie \
+                    port:py${python.version}-yaml \
+                    port:py${python.version}-sdnotify \
+                    port:py${python.version}-service_identity \
+                    port:py${python.version}-setproctitle \
+                    port:py${python.version}-setuptools \
+                    port:py${python.version}-treq \
+                    port:py${python.version}-txaio \
+                    port:py${python.version}-txtorcon \
+                    port:py${python.version}-u-msgpack-python \
+                    port:py${python.version}-urllib3 \
+                    port:py${python.version}-watchdog \
+                    port:py${python.version}-werkzeug \
+                    port:py${python.version}-zlmdb \
+                    port:py${python.version}-zopeinterface
+}
+
+subport crossbar {
+    name                crossbar
+    version             ${version}
```

Comment:
Setting a variable to itself does nothing.

Reply:
Removed